models/chainlink/avalanche_c/chainlink_avalanche_c_ccip_onramps.sql

```diff
@@ -0,0 +1,42 @@
+{{
+    config(
+        tags=['dunesql'],
+        schema = 'chainlink_avalanche_c',
+        alias = alias('ccip_onramps'),
+        materialized = 'incremental',
+        file_format = 'delta',
+        unique_key = ['evt_index', 'evt_tx_hash', 'message'],
+        post_hook='{{ expose_spells(\'["avalanche_c"]\',
+                                "project",
+                                "chainlink",
+                                \'["synthquest"]\') }}'
+    )
+}}
+
+ {% set avax_router = '0x27f39d0af3303703750d4001fcc1844c6491563c' %}
+ {% set test_date = "2022-06-01" %}
+
+select 
+'Avalanche Mainnet' as origin
+, 'Ethereum Mainnet' as destination
+, cast(json_extract_scalar(message, '$.sourceChainSelector') as INT256) as sourceChainSelector
```

Comment:
We may need to rename all column names in `lower_snake_case` format.
i.e.
`source_chain_selector`

Reply:
big oof
Running through this now